src/utils/extractOptions.js

```diff
@@ -1,8 +1,10 @@
+import _ from "lodash";
+
 const extractOptions = data => {
-  let options = [];
-  data.data.map(item => {
-    let campaign = {};
-    let channel = {};
+  const options = [];
+  _.flatMap(data.data, item => {
+    const campaign = {};
+    const channel = {};
     campaign["label"] = item.campaign;
     channel["label"] = item.channel;
     options.push(campaign);
```

Comment:
no need to `push`, just return it, see docs

Reply:
please resolve

Reply:
e3930bd24c19058065b931c37cb091594f1b1acb